controller/Previewer.php

```diff
@@ -193,23 +218,55 @@ public function getItem()
                     ->loadCompiledItemData();
 
                 $response['baseUrl'] = $itemPreviewer->getBaseUrl();
-
-            } else if ($itemUri) {
-                // Load RESOURCE item data
-                // TODO
+            } elseif ($itemUri) {
+                $item = $this->getResource($itemUri);
+                $lang = $this->getSession()->getDataLanguage();
+                $packer = new Packer($item, $lang);
```

Comment:
this looks tricky ..  we get packer -> we feed it with current service Locator,

Reply:
I think this is actually quite common in TAO. Definitly something to look at once we have propper DI setup ;)